src/Spryker/Client/RabbitMq/Model/Connection/ConnectionManager.php

```diff
@@ -73,8 +73,9 @@ public function __construct(
      */
     public function getChannelsByQueuePoolName(string $queuePoolName, ?string $localeCode): array
     {
+        $queuePools = $this->config->getQueuePoolsForStore(APPLICATION_STORE);
```

Comment:
Cover fallback logic, when we don't have a configured new application environment config.